elements/howto-tabs/howto-tabs.js

```diff
@@ -284,24 +311,81 @@
       // changes its role to `tab`.
       this.setAttribute('role', 'tab');
```

Comment:
No, for graceful degradation we are putting `role=heading` and then upgrading to `role=tab`